tempto-ldap/src/main/java/com/teradata/tempto/fulfillment/ldap/OpenLdapObjectDefinitions.java

```diff
@@ -31,31 +32,143 @@
     public static final LdapObjectDefinition TEST_GROUP =
             LdapObjectDefinition.builder("TestGroup")
                     .setDistinguishedName("cn=TestGroup,ou=Test,dc=tempto,dc=com")
-                    .setAttributes(testGroupAttributes())
+                    .setAttributes(getTestGroupAttributes())
                     .addObjectClasses(Arrays.asList("groupOfNames"))
-                            .build();
+                    .build();
+
+    public static final LdapObjectDefinition TEST_USER =
+            LdapObjectDefinition.builder("TestUser")
+                    .setDistinguishedName("uid=testuser,ou=Test,dc=tempto,dc=com")
+                    .setAttributes(getTestUserAttributes())
+                    .addObjectClasses(Arrays.asList("person", "inetOrgPerson"))
+                    .build();
```

Comment:
Remove it. Or if it is needed for tempto testing move it to `test` source root or to `tempto-examples`.